profiles/common/modules/features/multisite_business_indicators/multisite_business_indicators_standard/multisite_business_indicators_standard.module

```diff
@@ -163,13 +163,13 @@ function multisite_business_indicators_standard_views_data() {
 /**
  * Function for insert statistic into business_indicators_ext_index.
  *
- * @global object $user
- * @global object $language
- *
  * @param array $entity
  *   Insert into the db a business indicator record.
+ *
+ * @global object $user
+ * @global object $language
```

Comment:
I don't know why they are using these `@global` declarations but it is definitely never used in Drupal. They can be removed.